### PR TITLE
Adds error notifications to market page

### LINKED ISSUE #263: Show form errors on market page

### PR DESCRIPTION
**Pull request by bartaz**
Fixes #263 

## QA
- ./run
- go to market page of snap you own
- click on an icon and choose an image bigger than 256px
- click Apply
- error notification should appear

screen shot 2018-01-30 at 10 22 56

### CODE REVIEW

File: templates/publisher/market.html [hunk 1]
```diff
@@ -40,15 +40,23 @@ <h1 class="u-no-margin--top">{{ title }}</h1>
...
                   <div id="market-form-status" class="p-notification--{{ category }}" style="display: block">
                     <p class="p-notification__response">
                         {{ message }}
                     </p>
-                    </div>
+                  </div>
+                {% endfor %}
+
+                {% for error in error_list %}
+                  <div class="p-notification--negative" style="display: block">
```
Comment (anthonydillon) [inline templates/publisher/market.html]: @bartaz FYI the `display: block` can be removed when you update Vanilla to `v1.6.5`

**Comment by Greggless** [comment]
Can we get validation on the icon size before we click `Apply`?

**Comment by anthonydillon** [comment]
Sorry, mean't to remove `QA needed` label.

File: templates/publisher/market.html (hunk 1 above)
Comment (bartaz) [reply templates/publisher/market.html]: @anthonydillon Great, thanks.

**Comment by bartaz** [comment]
@Greggless Ideally yes, we should validate as much client side as possible. But the purpose of this branch was simply to show errors from API.

**Comment by Greggless** [comment]
Thanks @bartaz 

In that case +1 and we can address the validation at a later date

**Pull request merged**